cnf/Makegap.in

```diff
@@ -683,3 +674,15 @@ gapmpi.o: @srcdir@/system.h @srcdir@/gasman.h @srcdir@/objects.h \
           @srcdir@/vars.h @srcdir@/stats.h @srcdir@/gapmpi.h 
 	$(CC) -I. $(MPI_CFLAGS) -DCONFIG_H $(CFLAGS) -o gapmpi.o -c @srcdir@/gapmpi.c
 
```

Comment:
Why is there now two copies of this `Cygwin` code, both here and in `Makegap.bottom`?